### DISCUSSION TITLE
"Create desktop shortcut" with the name you've set already in RustDesk

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#7256] [Ideas]
ex:
If you created a shortcut for a device, it will create a shortcut with its `ID`, not the name you've set in RustDesk, you need to rename them manually if you create more than one shortcuts.

Another ideas from #7255, if this can come true, the shortcut's name needs to add some identity
ex:
Your remote device name is "My Macbook"

`My Macbook` for Remote Desktop
`My Macbook - File Transfer` for transfer files

But seems ` - File Transfer` is a little to long, maybe find another better way to identity?
Maybe add a small icon at the right corner of RustDesk icon? or both?

**Comment by southerndoc** [comment]
This would be awesome!  Wish Macs could create desktop shortcuts as well.
